Mathlib/Algebra/Order/Field/Power.lean

```diff
@@ -81,6 +81,8 @@ theorem zpow_lt_iff_lt (hx : 1 < a) : a ^ m < a ^ n ↔ m < n :=
   (zpow_strictMono hx).lt_iff_lt
 #align zpow_lt_iff_lt zpow_lt_iff_lt
 
+@[gcongr] alias ⟨_, zpow_lt_of_lt⟩ := zpow_lt_iff_lt
```

Comment:
```suggestion
@[gcongr] alias ⟨_, GCongr.zpow_lt_of_lt⟩ := zpow_lt_iff_lt
```

Reply:
I like the idea of moving auxiliary lemmas that exist only because `gcongr` can't handle `Monotone`/`StrictMono`/`Iff` lemmas to the `GCongr` namespace.